inputs/values/dockers.json

```diff
@@ -13,7 +13,7 @@
   "sv_base_docker": "us.gcr.io/broad-dsde-methods/gatk-sv/sv-base:2023-07-28-v0.28.1-beta-e70dfbd7",
   "sv_base_mini_docker": "us.gcr.io/broad-dsde-methods/vjalili/sv-base-mini:5994670",
   "sv_pipeline_base_docker": "us.gcr.io/broad-dsde-methods/gatk-sv/sv-pipeline:2023-09-13-v0.28.3-beta-af8362e3",
-  "sv_pipeline_docker": "us.gcr.io/broad-dsde-methods/gatk-sv/sv-pipeline:2023-09-13-v0.28.3-beta-af8362e3",
+  "sv_pipeline_docker": "us.gcr.io/talkowski-sv-gnomad/kveerara/sv-pipeline:kv_split_variants_8d7ca52",
```

Comment:
Thank you for working on this and updating the docker image. Most of our Docker images are built and updated automatically (i.e., when this PR is merged, the Docker images needing an update are automatically rebuilt, and this file is auto-updated, check [this commit](https://github.com/broadinstitute/gatk-sv/commit/8c9ddab01ab19b86f33afc01f5149645aeee0fa7) as an example); hence, you do not need to modify this file. We have also learned that changing this file can interfere with the automated process and has led to cases where inconsistencies are introduced in the docker images. 

I would suggest reverting changes to this file, and letting the automation build and update it. You can check how they are being built as part of a per-commit test under `Docker Images / Test Images Build`. 

For future reference, for accessibility reasons, we tend to keep the images under `broad-dsde-methods`.

Reply:
@kirtanav98 this file still needs to be reverted before this branch can be merged